packages/content/src/Application/PropertyResolver/Resolver/BlockPropertyResolver.php

```diff
@@ -53,14 +56,17 @@ public function resolve(mixed $data, string $locale, array $params = []): Conten
         \assert($metadata instanceof FieldMetadata, 'Metadata must be set to resolve blocks.');
         $metadataTypes = $metadata->getTypes();
 
+        /** @var TypedFormMetadata $typedFormMetadata */
+        $typedFormMetadata = $this->formMetadataLoader->getMetadata('block', $locale, []);
+        $globalBlocksMetadata = $typedFormMetadata->getForms();
         $contentViews = [];
         foreach ($data as $key => $block) {
             if (!\is_array($block) || !isset($block['type']) || !\is_string($block['type'])) {
                 continue;
             }
 
             $type = $block['type'];
-            $formMetadata = $metadataTypes[$type] ?? null;
+            $formMetadata = $globalBlocksMetadata[$type] ?? $metadataTypes[$type] ?? null;
```

Comment:
we prefer always the global block here now but what if a local block type was defined without a `ref` to a global block?